packages/amplication-e2e-tester/New project test/create.test.js

```diff
@@ -0,0 +1,38 @@
+const TITLE = "Amplication";
+const LOGIN_URL = "http://localhost:3001/login";
+const HOME_PAGE_URL = "http://localhost:3001/";
+const USER_EMAIL = "lisa@simpson.com";
+const USER_PASSWORD = "secret42";
+const APP_NAME = "my first app";
+const TIMEOUT=60000;
+var RESULT_URL;
+var RESULT_SPAN;
+describe("create new app test", () => {
+  beforeAll(async () => {
+    await page.goto(LOGIN_URL);
+  }, TIMEOUT);
+  it("should create new app  ", async () => {
+    page.setDefaultTimeout(TIMEOUT);
+    await expect(page.title()).resolves.toMatch(TITLE);
+    await (await page.waitForXPath("//input[@name='email']")).type(USER_EMAIL);
+    await (await page.waitForXPath('//input[@name="password"]')).type(

```

Comment:
Once the email input is rendered I think you assume the password input is ready as well.

Reply:
my problem is that waitForXPath returns an element so i can .click() it in the same line,
and $x returns an array of elements (in my case array with 1 element) and i dont know how to click on him without
creating a variable

Reply:
I see, then this is fine.